playwright.config.js

```diff
@@ -12,6 +12,16 @@ dotenv.config()
 /**
  * @see https://playwright.dev/docs/test-configuration
  */
+
+const webServerConfig = {
+  command: 'npm run build; npm start',
+  // Building the app can take some time:
+  timeout: 600_000,
+  port: 6060
+}
+
+const shouldStartWebServer = process.env.E2E_TEST_RUN_WEBSERVER !== "false"
```

Comment:
Wouldn't this cover all relevant situations? (Possibly also using it to set `reuseExistingServer`.)

```suggestion
const shouldStartWebServer = process.env.E2E_TEST_ENV === "local"
```

Reply:
i thought about this but i figured this would also help the cron job run faster if webserver build is not happening